wordpress-mobile/WordPress-iOS
# Primary Site Order

Fixes #2599

#### To test:
1. Launch WPiOS
2. Log into a WordPress.com with multiple blogs
3. Tap over My Sites Tab
4. Verify that the `Primary Blog` shows up on top

Needs review: @astralbodies 
Thanks in advance!
